lib/apr/payments.ex

```diff
@@ -3,17 +3,44 @@ defmodule Apr.Payments do
 
   alias Stripe.PaymentIntent
 
-  def liability_shift_happened(nil), do: false
+  def payment_info(external_id, "payment_intent") do
+    with {:ok, pi} <- PaymentIntent.retrieve(external_id, %{expand: ["payment_method"]}),
+         liability_shift <- liability_shift_from_payment(pi),
+         charge_data <- charge_data(pi) do
+      {:ok,
+       %{
+         liability_shift: liability_shift,
+         charge_data: charge_data,
+         card_country: pi.last_payment_error.payment_method.card.country,
+         cvc_check: pi.last_payment_error.payment_method.card.checks.cvc_check,
+         zip_check: pi.last_payment_error.payment_method.card.checks.address_postal_code_check,
```

Comment:
Do we have to worry that any of these sub-fields won't exist?

Reply:
i think we are safe but let me do a test and regardless add a defensive code there.

Reply:
ended up fixing this by first trying to get it from `last_payment_error` and if not trying to get it from `payment_method` and if not just return empty for all data.